packages/cli/src/__tests__/admin-api.test.ts

```diff
@@ -130,6 +140,96 @@ describe('admin api', () => {
     expect(getResultAfterDelete.models).toEqual([])
   })
 
+  it('admin pin API CRUD test', async () => {
+    const adminPinURLBaseString = `http://localhost:${daemon.port}/api/v0/admin/pins`
+
+    const fetchCode = async (): Promise<string> => {
+      return (await fetchJson(`http://localhost:${daemon.port}/api/v0/admin/getCode`)).code
+    }
+
+    // Get list of pins
```

Comment:
Just a nit, but I guess it would be cools to have separate tests for separate cases: list/single/delete/add